.github/workflows/build.yml

```diff
@@ -69,6 +69,9 @@ jobs:
       - run: opam exec -- git diff --exit-code
         if: ${{ matrix.ocaml-compiler == '4.12.0' }}
 
+      - run: opam exec -- git diff --exit-code
```

Comment:
This is duplicated. You should remove this diff

Reply:
Good catch. Updated!